docs/developers/v2/DEPLOYMENT.md

```diff
@@ -129,34 +129,40 @@ vault.setGovernance(ychad.eth)
    ```
 
 1. Monitor `harvest` and `tend` triggers for first few days. Call `harvest`/`tend` manually.
+1. If this is a new vault deployment, test deposit, withdraw, and transfer to ensure functionality is as expected.
```

Comment:
Style thing: should this be 2.?

Reply:
markdown automatically increments, I learned it from our docs hahaha https://yearn-docs-git-fork-marcoworms-strategy-checklist.yearn.farm/contributing/documentation/writing-style-guide

<img width="438" alt="Screen Shot 2022-10-14 at 10 48 40 AM" src="https://user-images.githubusercontent.com/7863230/195862958-5520b5de-9864-44bf-a451-23d220fc73d2.png">